config/locales/deed/deed-en.yml

```diff
@@ -6,29 +6,28 @@ en:
     collection_inactive: Collection Inactive
     collection_joined: Collection Joined
     deed:
-      html:
```

Comment:
No need to scope in .html anymore since we moved it to formats: [:html], it still looks at translations .deed instead of at .deed.html